### PR TITLE
Preliminary testing for future v0.2 release

### PR DESCRIPTION
**Pull request by vanderhe**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by vanderhe** [comment]
@conda-forge-admin, please rerender

File: recipe/build.sh [hunk 1]
```diff
@@ -1,7 +1,36 @@
...
+   ..
+)
 
-cmake $CMAKE_ARGS -GNinja -B_build
-cmake --build _build
-cmake --install _build
+mkdir -p _build
+pushd _build
+
+FFLAGS="-fno-backtrace" cmake "${cmake_options[@]}"
```
Comment (awvwgk) [inline recipe/build.sh]: Don't override the `FFLAGS` set by the environment but append new flags.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,46 +1,55 @@
...
 
 requirements:
   build:
-    - {{ compiler('c') }}
     - {{ compiler('fortran') }}
     - cmake
     - ninja
-    - pkg-config
   host:
+    - python
```
Comment (awvwgk) [inline recipe/meta.yaml:23]: Unless there are Python bindings with compiled extension modules, please keep a separate package for the binaries and the Python module to reduce the number of build artifacts. If preferred we can merge the sktools and skprogs feedstock and have multiple outputs here.

File: recipe/build.sh (hunk 1 above)
Comment (vanderhe) [reply recipe/build.sh]: What about now?

File: recipe/meta.yaml (hunk 2 above)
Comment (vanderhe) [reply recipe/meta.yaml:23]: okay

**Comment by vanderhe** [comment]
@awvwgk Any idea what's going on with e.g. `$PREFIX/bin/slateratom --version` segfaulting?

**Comment by awvwgk** [comment]
I see nothing which looks particular suspicious. Linking seems correct except for the Lapack/Blas overlinking, which wouldn't cause a segfault.

**Comment by vanderhe** [comment]
Statically linked library prevents segfaults when invoking the binaries. Not that this is a desired solution but might give some insight into the nature of the problem...

**Comment by vanderhe** [comment]
ctest refuses to run the build sanity test and issues "permission denied/BAD_COMMAND".

**Comment by vanderhe** [comment]
@conda-forge-admin, please rerender

**Comment by vanderhe** [comment]
@conda-forge-admin, please rerender

**Comment by vanderhe** [comment]
@conda-forge-admin, please rerender

**Pull request closed without merging**